proto/internal/temporal/server/api/historyservice/v1/request_response.proto

```diff
@@ -916,3 +917,11 @@ message CompleteNexusOperationRequest {
 
 message CompleteNexusOperationResponse {
 }
+
+message HealthCheckRequest {
+    string host_address = 1;
```

Comment:
this won't be used by the handler right?

Reply:
No. this is similar to the describe history pod api

Reply:
hmm yeah I guess it's mainly for making our client implementation to be easier since with this def it can figure the target host from the request itself.